static/js/src/ua-payment-modal.js

```diff
@@ -644,6 +669,11 @@ function handleCustomerInfoResponse(paymentMethod, data) {
   } else if (data.createdAt) {
     // payment method was successfully attached,
     // ask user to click "Pay"
+
+    if (currentTransaction.type == "purchase") {
+      checkoutEvent(analyticsFriendlyProducts(), "Added payment details");
```

Comment:
Steps should be integers.
```suggestion
      checkoutEvent(analyticsFriendlyProducts(), 2);
```

Reply:
```suggestion
      checkoutEvent(analyticsFriendlyProducts(), 2);
```